Userland/Libraries/LibCoredump/Reader.h

```diff
@@ -20,12 +20,20 @@ class Reader {
     AK_MAKE_NONMOVABLE(Reader);
 
 public:
-    static OwnPtr<Reader> create(const String&);
+    static OwnPtr<Reader> create(const StringView&);
```

Comment:
There's no need to pass StringViews by reference - see 8b1108e4858f797c9216dc8ae4a3918ad50c73b4

```suggestion
    static OwnPtr<Reader> create(StringView);
```